clients/drcachesim/simulator/caching_device_stats.cpp

```diff
@@ -126,6 +126,8 @@ caching_device_stats_t::print_counts(std::string prefix)
         std::setw(20) << std::right << num_hits << std::endl;
     std::cerr << prefix << std::setw(18) << std::left << "Misses:" <<
         std::setw(20) << std::right << num_misses << std::endl;
+    std::cerr << prefix << std::setw(18) << std::left << "Invalidations:" <<
```

Comment:
We have a number of "golden output" tests that this will break without also updating their expected output: see the CI test failures

Reply:
Fixed.